dspace-api/src/main/java/org/dspace/app/requestitem/RequestItemSubmitterStrategy.java

```diff
@@ -22,6 +24,7 @@
  * @author Andrea Bollini
  */
 public class RequestItemSubmitterStrategy implements RequestItemAuthorExtractor {
+    private static final Logger LOG = LogManager.getLogger();
```

Comment:
This logger doesn't appear to be used anywhere in this file. Maybe it should just be removed until it's needed?

Reply:
Thanks, already noted, now fixed.